types/mock-req-res/mock-req-res-tests.ts

```diff
@@ -2,7 +2,7 @@ import { Request, RequestHandler, Response } from "express";
 import { mockRequest, mockResponse } from "mock-req-res";
 
 const handler: RequestHandler = (req: Request, res: Response) => {
-    return res.status(200).json(`Hello from handler with an originalUrl value of '${req.originalUrl}'`);
+    return void res.status(200).json(`Hello from handler with an originalUrl value of '${req.originalUrl}'`);
```

Comment:
This `return` and many others are entirely optional, but are kept as `return void` for demonstration purposes.